src/Framework/test/Microsoft.AspNetCore.App.UnitTests.csproj

```
@@ -3,8 +3,11 @@
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
     <RootNamespace>Microsoft.AspNetCore</RootNamespace>
     <VerifyAncmBinary Condition="'$(TargetOsName)' == 'win' AND '$(TargetArchitecture)' != 'arm'">true</VerifyAncmBinary>
+    <!-- Only verify assembly versions in non-dev builds. Dev builds always produce assembly version `42.42.42.42` -->
+    <VerifyPackageAssemblyVersions Condition="'$(OfficialBuild)' == 'true' or '$(DotNetUseShippingVersions)' == 'true'">true</VerifyPackageAssemblyVersions>
     <TestDependsOnAspNetAppPackages>true</TestDependsOnAspNetAppPackages>
     <SkipHelixArm>true</SkipHelixArm>
+    <PackagesLayoutRoot>$([MSBuild]::NormalizeDirectory('$(DOTNET_ROOT)', '$(ArtifactsObjDir)', 'Packages.Layout', '$(Configuration)'))</PackagesLayoutRoot>
```

How does this work when both `$(DOTNET_ROOT)` and `$(ArtifactsObjDir)` are absolute paths❔ I also don't recommend using `$(DOTNET_ROOT)` for layouts because it'll be readonly in some Docker scenarios.
```suggestion
    <PackagesLayoutRoot>$([MSBuild]::NormalizeDirectory('$(ArtifactsObjDir)', 'Packages.Layout', '$(Configuration)'))</PackagesLayoutRoot>
```